docs/topics/mta-web-viewing-an-assessment-report.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-reviewing-an-analysis-report_{context}"]
+= Viewing an assessment report
+
+An {ProductShortName} assessment report displays the aggregated assessement of the data obtained from multiple questionnaires for multiple applications.
+
+.Procedure
+
+. To view an assessment report, in *Migration* view, in the menu on the left side of the window, click *Report*.
++
+The aggregated assessment report of all aplications is displayed.
++
+image::mta-assessment-report.png[Assessment report]
++
+* To display a report on the data from a particular questionnaire, select the required questionnaire from a drop-down list of all questionnaires in the *Current landscape* pane of the report. By default, all questionnaires are selected.
+* In the *Identified risks* pane of the report, you can sort the displayed list by application name, level of risk, questionnaire, questionnaire section, question, and answer.
+* To display a report for a specific application:
+.. Click the link in the *Applications* column.
++
+The applications included in the link are displayed as a list.
+.. Click the required application.
```

Comment:
And Here we about Click the required application on **Application Inventory** page

Reply:
@kpunwatk 
But then we could get there directly from the 'Application inventory' page? Without going through 'Reports'? You said before "No option in Application inventory to navigate to Report's Tab/page for Assessment directly". What am I missing?

Reply:
Here b.Click the required application, we are talking about Application Inventory which opens the side drawer and in Details see Application risk(as shown in attached below screenshot) and Reviews Tab, right?

Yes, Through Application Inventory we can' t directly navigate to Reports page. Just can see Assessment **Reviews** Tab and **Application risk**(in Details).